config/config.yaml.dist

```diff
@@ -267,8 +267,27 @@ dex:
 #
 #        charts:
 #            clusterAutoscaler:
-#                chart: "banzaicloud-stable/cluster-autoscaler"
-#                version: "0.12.3"
+#                chart: "stable/cluster-autoscaler"
+#                version: "6.2.0"
+#                imageVersions:
```

Comment:
Why not a map instead?

Reply:
Could be a map, but you have to iterate it over anyway, to match all constraints.